generator.json

```
@@ -46,6 +48,7 @@
       },
       "ChangeNamespace": {
         "Mappings": {
+          "TerraFX.Interop.Windows(.*)": "Silk.NET.Win32$1",
           "TerraFX.Interop(.*)": "Silk.NET$1"
```

It has occurred to me that ChangeNamespace currently does not take into account ordering of entries in here or otherwise uses the longest match. As such, this should either be:
```suggestion
          "TerraFX.Interop.Windows(.*)": "Silk.NET.Win32$1",
          "TerraFX.Interop.(?!Windows)(.*)": "Silk.NET$1"
```
Or we should figure out how to respect ordering.

---

this was addressed

---

Ah yeah you're right, need to remember that right clicking on the link above the comment doesn't do what I think it does!